texttospeech/test/quickstart.test.mjs

```diff
@@ -0,0 +1,42 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import fs from 'fs/promises';
+import { existsSync } from 'node:fs';
+import { describe, it } from 'mocha';
```

Comment:
suggestion: Not sure if you're looking to experiment, but it may be worth checking out the newer [built-in test runner](https://nodejs.org/api/test.html). I'm not sure if it has all the reporting capabilities we need, but if it's a path to reducing the need for mocha and chai that could be nice.

Reply:
oh cool; thanks for pointing this out.

Reply:
I'll revisit this later, but for now will go ahead and leave these as is